ControllerKey.js

```diff
@@ -307,7 +309,7 @@ export class ControllerKey {
    * via `fromSecret` with `cache` set to `true` in order to ensure the key
    * cannot be loaded via `fromCache`.
    *
-   * @param {Object} options - The options to use.
+   * @param {object} options - The options for the caches.
```

Comment:
I don't understand "for the caches"... seems the previous was better even if vague. Did you mean "for the cache"? Even then I'm not sure what that means. Also, everything else is "The options to use." so let's keep it the same for easier maintenance.

Reply:
This was reverted to `The options to use.` here: https://github.com/digitalbazaar/web-kms-client/pull/2/commits/48dd98df86624e1849176410812202c047112ec3